src/openforms/submissions/report.py

```diff
@@ -15,6 +16,35 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class CoSigner:
+    co_sign_data: dict
+
+    @property
+    def representation(self) -> str:
+        return self.co_sign_data.get("representation", "")
+
+    @property
+    def identifier(self) -> str:
+        return self.co_sign_data.get("identifier", "")
+
+    @property
+    def auth_attribute(self) -> str:
+        return self.co_sign_data.get("co_sign_auth_attribute", "")
```

Comment:
this should use the human readable label of the `AuthAttributes` choices instead of the raw string value:

```
class AuthAttribute(models.TextChoices):
    """
    core identifying attributes retrieved from the authentication plugins
    """

    bsn = "bsn", _("BSN")
    kvk = "kvk", _("KvK number")
    pseudo = "pseudo", _("Pseudo ID")
    employee_id = "employee_id", _("Employee ID")
```

there's some sort of `AuthAttribute.labels` helper available, check the django docs :)